demisto_sdk/commands/create_id_set/tests/update_id_set_test.py

```diff
@@ -47,6 +47,12 @@ def test_create_id_set_output(self):
         id_set_creator.create_id_set()
         assert os.path.exists(self.file_path)
 
+    def test_create_id_set_on_specific_pack(self):
+        id_set_creator = IDSetCreator(self.file_path, input='Packs/AMP')
+
+        id_set_creator.create_id_set()
+        assert os.path.exists(self.file_path)
```

Comment:
add more checks here:
* the pack elements were collected
* there aren't any other element (from other packs)

Reply:
@ChanochShayner 
what about this case
> there aren't any other element (from other packs)